src/main/environment/common_ci.properties

```diff
@@ -98,3 +98,5 @@ eSanjeevani.routeUrl: @env.ESANJEEVANI_ROUTE_URL@
 biometric.discover.url = @env.BIOMETRIC_URL@
 biometric.deviceInfo.url = @env.BIOMETRIC_URL@/rd/info
 biometric.capture.url = @env.BIOMETRIC_URL@/rd/capture
+eAusadhaUrl=https://dlc.kar.nic.in/e-services/api/DWInstituteInward
+authorization=Basic a3Ntc2NsOktzbXNjbCMxMjM=
```

Comment:
Can you please make this key more specific? "authorization" is not explanatory at application level.
"eausadha-authorization" maybe?

Reply:
Also, can we please remove this key from committing to public?

Reply:
resolved the conflict and modified these properties
please review and merge it @drtechie  @ravishanigarapu